src/plugins/ncr/components/labels-picker/index.jsx

```diff
@@ -49,14 +49,15 @@ class Labels extends React.PureComponent {
     this.handleChange = this.handleChange.bind(this);
   }
 
-  UNSAFE_componentWillReceiveProps({ node }) {
+  componentDidUpdate({ node }) {
     if (!node) {
       return;
     }
     const { touched } = this.state;
     if (touched) {
       return;
     }
+    // eslint-disable-next-line react/no-did-update-set-state
     this.setState({
       selected: node.get('labels', []),
```

Comment:
@rsumilang do you want to check if the labels have changed before re-setting them into state?